h2o-py/h2o/h2o.py

```diff
@@ -2120,6 +2122,41 @@ def upload_mojo(mojo_path):
     return mojo_estimator
 
 
+def print_mojo(mojo_path, format="json", tree_index=None):
+    """
+    Generates string representation of an existing MOJO model. 
+    :param mojo_path:  Path to the MOJO archive on the user's local filesystem
+    :param format:  Output format. Possible values: json (default), dot 
+    :param tree_index:  Index of tree to print (only work dot format)
+    :return: An string representation of given MOJO in given format
+    """    
+    assert_is_type(mojo_path, str)
+    assert_is_type(format, str, None)
+    assert_satisfies(format, format in [None, "json", "dot"])
+    assert_is_type(tree_index, int, None)
+
+    ls = H2OLocalServer()
+    jar = ls._find_jar()
+    java = ls._find_java()
+    if format is None:
+        format = "json"
+    cmd = [java, "-cp", jar, "hex.genmodel.tools.PrintMojo", "--input", mojo_path, "--format", format]
+    if tree_index is not None:
+        cmd += ["--tree", str(tree_index)]
+    output_file = tempfile.mkstemp(prefix="mojo_output")[1]
+    try:
+        with open(output_file, 'w+') as stdout:
+            return_code = subprocess.call(cmd, stdout=stdout)
+            stdout.seek(0)
+            output = stdout.read()
+    except OSError as e:
+        traceback = getattr(e, "child_traceback", None)
+        raise H2OError("Unable to print MOJO: %s" % e, traceback)
```

Comment:
you don't need to handle Py2.7 special case here ("child_traceback").
Official supports ends in 3 weeks...

Reply:
https://pythonclock.org/

Reply:
yes, but accoriding to @michalkurka our customers might be stuck on py2 for some more time